docs/rules/no-implicit-coercion.md

```diff
@@ -33,10 +33,11 @@ This rule is aimed to flag shorter notations for the type conversion, then sugge
 
 This rule has three main options and one override option to allow some coercions as required.
 
-* `"boolean"` (`true` by default) - When this is `true`, this rule warns shorter type conversions for `boolean` type.
-* `"number"` (`true` by default) - When this is `true`, this rule warns shorter type conversions for `number` type.
-* `"string"` (`true` by default) - When this is `true`, this rule warns shorter type conversions for `string` type.
-* `"allow"` (`empty` by default) - Each entry in this array can be one of `~`, `!!`, `+` or `*` that are to be allowed.
+-   `"boolean"` (`true` by default) - When this is `true`, this rule warns shorter type conversions for `boolean` type.
+-   `"number"` (`true` by default) - When this is `true`, this rule warns shorter type conversions for `number` type.
+-   `"string"` (`true` by default) - When this is `true`, this rule warns shorter type conversions for `string` type.
+-   `"disallowTemplateShorthand"` (`false` by default) - When this is `true`, this rule warns shorter type conversions using template strings.
```

Comment:
```suggestion
-   `"disallowTemplateShorthand"` (`false` by default) - When this is `true`, this rule warns `string` type conversions using `${expression}` form.
```

Or something like this, to be more specific. "shorter type conversions using template strings" could be interpreted as ``` `` + foo ```, which is also warned but unrelated to this option.

Can you also switch the order of `"disallowTemplateShorthand"` and `"allow"` in this list (as well as the order of sections below), so that `"disallowTemplateShorthand"` becomes last? `"allow"` is related to the previous three options, while `"disallowTemplateShorthand"` is an option for itself.